packages/salesforcedx-utils/test/jest/cli/cliCommandExecution.test.ts

```diff
@@ -0,0 +1,203 @@
+import { CancellationToken, CliCommandExecution, Command } from '../../../src';
+import {
+  CANCELATION_INTERVAL,
+  KILL_CODE,
+  NO_PID_ERROR,
+  NO_STDERR_ERROR,
+  NO_STDOUT_ERROR
+} from '../../../src/cli/cliCommandExecution';
+import { Observable } from 'rxjs/Observable';
+import * as kill from 'tree-kill';
+jest.mock('tree-kill');
+
+const treeKillMocked = jest.mocked(kill);
```

Comment:
This is how you correctly type the mocked function for use in the test while keeping typescript happy.